src/test/java/registrationTest.java

```diff
@@ -0,0 +1,36 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class registrationTest extends BaseTest {
+
+    @Test
+    public static void registrationNavigation() {
```

Comment:
Please, add the class in TestNG.xml

For example

![image](https://user-images.githubusercontent.com/104761482/221346570-1653c4ce-12a3-41a3-a812-c8beeab30e39.png)